addons/google_account/controllers/main.py

```diff
@@ -1,12 +1,15 @@
+# -*- coding: utf-8 -*-
+
 import simplejson
 import urllib
+from werkzeug.exceptions import BadRequest
+import werkzeug.utils
+
 import openerp
-from openerp import http
-from openerp.http import request
+from openerp.addons.web import http
+from openerp.addons.web.http import request
```

Comment:
The canonical location is `openerp.http`, `openerp.addons.web.http` is an alias for backwards compatibility.